operations/filter_kgraph_continuous_attribute.yml

```diff
@@ -0,0 +1,48 @@
+id: filter_kgraph_continuous_attribute
+subclass_of: filter_kgraph
+name: filter kgraph (continuous attribute)
+description: This operation removes kgraph edges based on the value of a continuous edge attribute. Edges without the given attribute are left alone.
+examples:
+- input: fill_and_filter/messages/09_filtered_kgraph_attribute_input.json
+  output: fill_and_filter/messages/10_filtered_kgraph_continuous_attribute_output.json
+  parameters:
+    edge_attribute: normalized_google_distance 
+    threshold: 0.3
+    remove_above: true
+    qnode_keys: 
+      - drug
+input_requirements: []
+output_guarantees:
+- no edges in the kgraph which have attribute values that are below/above the given value.
+allowed_changes:
+- remove knodes
+- remove kedges
+parameters:
+  type: object
+  properties:
+    edge_attribute:
+      description: The name of the edge attribute to filter on.
+      type: string
+      example: normalized_google_distance
+    threshold:
+      description: The value to compare attribute values to.
+      type: float
+      example: 1.2
+    remove_above:
+      description: Indictes whether to remove above or below the given threshold.
+      type: bool
+      example: true
+    edge_keys:
+      description: This indicates if you only want to remove edges with specific edge_keys. If not provided or empty, all edges will be filtered on.
+      type: array
+      example: [e01]
+      default: null
+    qnode_keys:
+      description: This indicates if you only want nodes corresponding to a specific list of qnode_keys to be removed. If not provided or empty, no nodes will be removed when filtering.
+      type: array
+      example: [n01]
+      default: null
```

Comment:
The default `null` is not an allowed value for `type: array`. I don't think this makes the JSON Schema instance invalid, but it's definitely undesirable. If we intend `null` to be an allowed value, we can do that. But we need to discuss how. JSON Schema allows `type: [array, null]`, but OpenAPI 3.0.x does not, and uses `nullable: true` instead. OpenAPI 3.1 fixes this and reverts to the JSON Schema behavior. My preference is to use JSON Schema 2020-12/OpenAPI 3.1 and lists of types.

Reply:
@patrickkwang I forgot what we ended up deciding to do here. Should I change this to a list of types (null and array)? Or did we not come to a decision on this?